### PR TITLE
Add taxadb python package

### PR DESCRIPTION
**Pull request by gtonkinhill**
Adds the taxadb python package which helps to locally query the ncbi taxonomy

### CODE REVIEW

File: recipes/taxadb/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{% set name = "taxadb" %}
+{% set version = "0.12.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
```
Comment (johnlees) [inline recipes/taxadb/meta.yaml:9]: Why the `name[0]`, is that common?

File: recipes/taxadb/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    - taxadb --help
+
+about:
+  home: "https://github.com/HadrienG/taxadb"
+  license: MIT
+  license_family: MIT
+  license_file:
+  summary: "locally query the ncbi taxonomy"
+  doc_url:
+  dev_url:
```
Comment (johnlees) [inline recipes/taxadb/meta.yaml:45]:
```suggestion
```

File: recipes/taxadb/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  imports:
+    - taxadb
+  commands:
+    - taxadb --help
+
+about:
+  home: "https://github.com/HadrienG/taxadb"
+  license: MIT
+  license_family: MIT
+  license_file:
```
Comment (johnlees) [inline recipes/taxadb/meta.yaml:42]:
```suggestion
```

File: recipes/taxadb/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  summary: "locally query the ncbi taxonomy"
+  doc_url:
+  dev_url:
+
+extra:
+  recipe-maintainers:
+    -gtonkinhill
```
Comment (johnlees) [inline recipes/taxadb/meta.yaml:49]: This is a conda-forge thing
```suggestion
```

File: recipes/taxadb/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: eca577cd0f4ff2f26c7d8f68d7f5a0d2f7a2f73e623139ded2493e429b23cecd
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - taxadb = taxadb.app:main
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (johnlees) [inline recipes/taxadb/meta.yaml:17]:
```suggestion
  script: "{{ PYTHON }} -m pip install --no-deps --ignore-installed --no-cache-dir . -vv"
```

File: recipes/taxadb/meta.yaml (hunk 4 above)
Comment (gtonkinhill) [reply recipes/taxadb/meta.yaml]: woops, should not have left it to the skeleton script!

File: recipes/taxadb/meta.yaml (hunk 1 above)
Comment (gtonkinhill) [reply recipes/taxadb/meta.yaml:9]: This was suggested by the conda skeleton script. It is probably okay either way so happy to go with your preference

**Review by johnlees — APPROVED** [review]

**Pull request merged**